docs/_layouts/default.html

```diff
@@ -56,7 +56,8 @@
             </ul>
           </li>
           <li>
-            <span class="nav-header">Bolt Developer Updates</span>
+            <span class="nav-header">
+              <a href="{{ '/blog.html' | relative_url }}" class="center-vertical">Bolt Developer Updates</a></span>
```

Comment:
The `.html` part doesn't seem to be necessary. I'm not sure I understand why though...

Reply:
Weird. Updated and also removed `.html` from the link to the debugging page while I was at it.